### DISCUSSION TITLE
苍角，到

### DISCUSSION THREAD

**Discussion by biliqide** [share121/inter-knot#250] [其他]
小苍角，外面蓝蓝的，里面粉粉的
![Screenshot_2024-08-09-14-09-19-42_149003a2d400f6adb210d7e357a3a646](https://github.com/user-attachments/assets/18d6552d-9bd1-410d-8571-4bf85a3567c7)

**Comment by Mu-7593** [comment]
小蓝娘可爱的捏

**Comment by xqd01** [comment]
可爱香草

**Comment by sabrin1145** [comment]
![微信图片_20240809232548](https://github.com/user-attachments/assets/5b46b38f-10ff-446a-bf9f-94cd28597887)

**Comment by jiahao995** [comment]
可爱捏

**Comment by TraitYoung** [comment]
我是对空六课的，请你和我走一趟_
